### DISCUSSION TITLE
Hit a hundred meshes, obligatory thank you

### DISCUSSION THREAD

**Discussion by Foxtrot-Oscar** [cnr-isti-vclab/meshlab#1424] [Show and tell]
Meshlab has been a fantastic little tool in my starting a 3D printing service/business. I have probably saved hundreds of dollars of potentially wasted material in the process already in the past few months, thanks to this app being in the pipeline. Great job guys!

**Comment by donbright** [comment]
me too, thought i'd continue this one instead of starting a new thread.

 besides using it to test openscad back when i worked on that, now im trying to use it to create rust code to build the Johnson Solids, right now the Rotunda, hopefully for submission to the https://github.com/virtualritz/polyhedron-ops project. The labels help alot debugging my face indexes.

![Screenshot from 2023-10-03 22-45-46](https://github.com/cnr-isti-vclab/meshlab/assets/896858/09511ecb-14a3-4548-9ebc-bafea153de3b)

thanks

**Comment by methot-ilya-3ds** [comment]
Also want to shout out here! Meshlab has been my go-to for testing out objects quickly and efficiently. Great for converting and optimising meshes. Thanks so much MeshLab!
